crates/aptos/src/common/types.rs

```diff
@@ -994,6 +994,11 @@ pub fn load_account_arg(str: &str) -> Result<AccountAddress, CliError> {
         })
     } else if let Ok(account_address) = AccountAddress::from_str(str) {
         Ok(account_address)
+    } else if let Some(Some(account_address)) =
+        CliConfig::load_profile(Some(str), ConfigSearchMode::CurrentDirAndParents)?
+            .map(|p| p.account)
+    {
+        Ok(account_address)
```

Comment:
This works great if it's in the CLI.

I do have one more extension to this, which will resolve another GitHub issue, and get you deeper into the weeds of address rotation: https://github.com/aptos-labs/aptos-core/issues/6295

Take a look at `aptos account lookup-address`

Reply:
Similarly, this would also be good to do account management:
* If my private key exists, and the address that I see locally has a different auth key, I can provide a useful message saying the private key isn't supported for the account, it might be rotated or wrong.
* This kind of support will allow for future management of keys in the CLI

Reply:
Sounds good!

For account management mentioned here, is this part of the Q2 item "Multisig account /account management"?